packages/dara-core/dara/core/cli.py

```diff
@@ -74,6 +75,7 @@ def start(
     debug: Optional[str],
     log: Optional[str],
     reload_dir: Optional[List[str]],
+    skip_jsbuild:bool,
```

Comment:
Strange, do we not have the format checker running in CI?

Reply:
We don't have one set up for Dara